src/test/java/seleniumWebDriver/AllLocators.java

```diff
@@ -0,0 +1,92 @@
+package seleniumWebDriver;
+
+import org.junit.Assert;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class AllLocators {
```

Comment:
this class contains duplicating steps and validations for login
let's create class Locators which will contain By constants, it could be useful for LoginTest